### ISSUE TITLE
Rasperry Pi - ArchLinux: game config missing

### ISSUE THREAD

**Issue by shenron** [DPRCZ/gpsp#8]
Hi,

I compile successfully gpsp on my Raspberry pi / Archlinux.
In my gpsp folder I have 

/home/pi/gpsp/raspberrypi/
- game_config.txt
- gba_bios.bin
- gpsp
- keys.txt

I run ./gpsp and when I launch a game.gba I have this error with a white screen:

> audio: freq 0, size 0
> game config missing

Thank you for help.

**Comment by shenron** [comment]
I'm sorry, in fact GPSP is extremely slow ...
The white screen it's a 'loading screen', after one minute the game start.
But it's very very slow. Do you think gpSP run on a Raspberry pi B rev 1 with 256 ram ?

**Comment by DPRCZ** [comment]
I originaly complied gpSP for Raspberian on Rpi B rev 1. And it works nearly fullspeed with pokemon leaves.
  ----- Original Message -----

**Comment by shenron** [comment]
Yes, I just test with Raspbian. It works ... But with Arch it's so slow ... :(
